docs/Github-workflow.md

```diff
@@ -0,0 +1,69 @@
+- Change the Project column to **In Progress**
+- Remove the **Needs Dev** label, if applicable
+
+## Needs Discussion
+Many issues require discussion and definition before development begins. For example, multiple approaches may be considered and the community should discuss them before a contributor proceeds with development. For those issues, add the **Needs Discussion** label and raise the issue in the weekly performance chat.
+
+## Needs Decision
+After discussion, a formal vote may be needed to determine how to proceed. If that’s the case, please tag @bethanylang in the issue for assistance with setting up a vote via GitHub comment. [An example vote can be found here](https://github.com/WordPress/performance/issues/92#issuecomment-1068215411).
+
+## Needs Dev
+If an issue requires (more) development and you are unable to complete it yourself, remove yourself as the assignee and add a **Needs Dev** label.
+
+## Needs Testing
+If you have completed initial engineering and want community members to test prior to merge, add the Needs Testing label.
+
+# Pull requests
+All pull requests must:
+
+- Be associated with an issue
+- Have [Type] and [Focus] label matching the related issue’s [Type] and [Focus] labels
+- Have either a milestone or the “no milestone” label
+
+When a PR for an issue is ready for review:
+- Change the Project column on the issue to **Review**
+- Add the **Needs Review** label to the issue
+
+Reviewers will be auto-assigned based upon the issue’s assigned Project. Note that all PRs require at least two reviewers.
+
+## Trac tickets
+[Trac](https://core.trac.wordpress.org/) is the system used to track issues related to WordPress core. There is a [‘performance’ label in Trac](https://core.trac.wordpress.org/query?status=!closed&focuses=~performance) that has been added to over 300 issues dating back nearly 15 years. 
+
+While the Performance team’s focus is on the work in the GitHub repo, the community is welcome to consult with the team on performance-related Trac issues. There may also be issues in Trac that are duplicates of planned work in the GitHub repo, or warrant further discussion for inclusion in the plugin.
+
+Initial discussion on performance-related Trac issues should take place in the Trac issue. When an issue is ready to begin development:
+
+In GitHub:
+- If it is a smaller issue or patch, no need to add to GitHub; manage in Trac
+- If it is a larger issue, create a new GitHub issue for it in the Performance repo, following the workflow outlined in this document
+- Include a link to the related Trac issue in the issue description
+
+In Trac:
+- Add a link to the related GH issue
+- Change the status to assigned
+Change the Owner to yourself
```

Comment:
I think this section is still a bit preliminary. There's more complexity to this, it really depends on each individual Trac ticket whether it is something worth (or even _possible_) to implement in the Performance Lab plugin, or whether it needs to be implemented against WP core immediately.

I wonder if we should for now keep the "Trac tickets" section limited to the brief introduction above this marked section and define a proper workflow around Trac separately. We can later expand this doc as needed. I think that would allow us to get all the rest of it which has been clear for a while merged quickly.